### PR TITLE
Add the DOS scanner wrapper

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: plugins/scanners/dos/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      DOS:
+        options:
+          url: '${DOS_SERVER_URL}'
+          frontendUrl: '${DOS_FRONTEND_URL}'
+        secrets:
+          token: '${DOS_SERVER_TOKEN}'
+```
+
+where `DOS_SERVER_URL`, `DOS_FRONTEND_URL` and `DOS_SERVER_TOKEN` are environment variables that are set to the respective values.
+The `DOS_FRONTEND_URL` is meant to points to the cleanrance UI which can be used to inspect scan results and to create license findings curations for use with the [DOS Package Configuration Provider].
```
Comment (Etsija) [inline plugins/scanners/dos/README.md:23]: ..."is meant to point to the Clearance UI"...

File: plugins/scanners/dos/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Double Open Server (DOS) Scanner Wrapper
+
+The DOS scanner wrapper is a client for [DOS] to initiate scans on the server-side, performed by [ScanCode], and to reuse scan results on a per-file basis.
+This effective implements "delta-scanning" of only the changed files between package versions.
```
Comment (mnonnenmacher) [inline plugins/scanners/dos/README.md:4]: "effectively"
As the scanner uses its own storage, should this mention that no other storage writer should be configured?

File: plugins/scanners/dos/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Usage
+
+The DOS scanner wrapper requires [DOS] to be running on a host and corresponding `DosScannerConfig` to be present in ORT's `config.yml`, e.g.
```
Comment (mnonnenmacher) [inline plugins/scanners/dos/README.md:8]: "**a** corresponding"
Replace ", e.g." with ":"

File: plugins/scanners/dos/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    config:
+      DOS:
+        options:
+          url: '${DOS_SERVER_URL}'
+          frontendUrl: '${DOS_FRONTEND_URL}'
+        secrets:
+          token: '${DOS_SERVER_TOKEN}'
+```
+
+where `DOS_SERVER_URL`, `DOS_FRONTEND_URL` and `DOS_SERVER_TOKEN` are environment variables that are set to the respective values.
```
Comment (mnonnenmacher) [inline plugins/scanners/dos/README.md:22]: drop "where"

File: plugins/scanners/dos/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+The `DOS_FRONTEND_URL` is meant to point to the Clearance UI which can be used to inspect scan results and to create license findings curations for use with the [DOS Package Configuration Provider].
```
Comment (mnonnenmacher) [inline plugins/scanners/dos/README.md:23]: "findings" -> "finding"

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: plugins/scanners/dos/README.md (hunk 2 above)
Comment (sschuberth) [reply plugins/scanners/dos/README.md:4]: The implementation hard-codes `readFromStorage = false, writeToStorage = false`, so I've added a sentence about that.

**Review by mnonnenmacher — APPROVED** [review]

**Comment by sschuberth** [comment]
The Docker Build failure is unrelated.

**Pull request merged**